tiproxy/tiproxy-overview.md

```diff
@@ -59,16 +59,16 @@ TiProxy 不适用于以下场景：
 
 ### 部署 TiProxy
 
-1. 生成自签名证书。
+1. 对于 TiUP v1.15.0 以下的版本，需要手动生成自签名证书。
```

Comment:
Currently, TiUP v1.15.0 is not released. If v1.15.0 is released before v8.0, the current description will remain accurate. Otherwise, we should update this to "对于 TiUP v1.14 及以下的版本"